### PR TITLE
#4: Solving Median of two sorted Arrays problem

### PR DESCRIPTION
**Pull request by DShydlouski**

### CODE REVIEW

File: ScnSoft-Education/LeetCodeTraining.Tests/MedianProblemTests.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+using LeetCodeTraining.MedianOfTwoSortedArrays;
+
+namespace LeetCodeTraining.Tests
+{
+    public class MedianProblemTests
+    {
+        [Theory]
+        [InlineData((int[])[1, 3], (int[])[2], 2)]
+        [InlineData((int[])[1, 2], (int[])[3, 4], 2.5)]
+        public void MedianProblemSolutionTest(int[] num1, int[] num2, double expected)
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining.Tests/MedianProblemTests.cs:10]: what does't mean num1, num2?
Don't use magic parameter names
Basically array != number

File: ScnSoft-Education/LeetCodeTraining/MedianOfTwoSortedArrays/MedianOfTwoSortedArraysSolution.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+            if (isEven)
+            {
+                result = (buffResult[middle - 1] + buffResult[middle]) / 2D;
+                return result;
+            }
+            else
+            {
+                result = buffResult[middle];
+                return result;
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining/MedianOfTwoSortedArrays/MedianOfTwoSortedArraysSolution.cs:33]: you can "return result;" once in the end of method

File: ScnSoft-Education/LeetCodeTraining.Tests/MedianProblemTests.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+using LeetCodeTraining.MedianOfTwoSortedArrays;
+
+namespace LeetCodeTraining.Tests
+{
+    public class MedianProblemTests
+    {
+        [Theory]
+        [InlineData((int[])[1, 3], (int[])[2], 2)]
```
Comment (dsubotkovsky) [inline ScnSoft-Education/LeetCodeTraining.Tests/MedianProblemTests.cs:8]: add unit test when arrays are empty (or one of it)
